openshift/4.0/pipelines/Jenkinsfile.cicd

```diff
@@ -42,6 +42,10 @@ pipeline {
     ROCKET_CHAT_SECRET = 'rocket-chat-secrets'
     ROCKET_CHAT_WEBHOOK_KEY = 'rocket-deploy-webhook'
 
+    // MS Teams Configuration
+    MS_TEAMS_SECRET = 'ms-teams-secrets'
+    MS_TEAMS_WEBHOOK_KEY = 'notification-webhook'
+
```

Comment:
This needs new secrets for MS Teams. I already created it for PSP.
The documentation explains how to configure for new projects.